src/marketing/search-terms-add.md

```diff
@@ -36,8 +36,34 @@ _Search Terms_
 
 ## Delete a Search Term
```

Comment:
We should probably fix capitalization in other parts of page too. "search term" is not really a proper noun and subheadings should be sentence case.

```suggestion
## Delete a search term
```